students/rod_musser/lesson01/front3.py

```diff
@@ -0,0 +1,3 @@
+def front3(str):
+    shortStr = str[0:3]
```

Comment:
make sure to stay consistent with python naming conventions, this should be snake_case